spec/jobs/deposit_status_job_spec.rb

```diff
@@ -7,6 +7,15 @@
 
   let(:instance) { described_class.new }
 
+  context 'with a not found druid' do
+    # An intentionally invalid druid so it does not collide with any test objects
+    let(:message) { '{"druid":"druid:aa11ii1111"}' }
+
+    it 'acks the message (and does not raise)' do
+      expect(run).to eq(:ack)
+    end
+  end
+
```

Comment:
And here's a test that the above change fixes, which corresponds to the HB alert we were seeing in production